entrypoint.sh

```diff
@@ -0,0 +1 @@
+poetry run gunicorn --bind 0.0.0.0:$PORT "todo_app.app:create_app()" 
```

Comment:
Recall that it's a good habit to always include a [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) at the top of a script
```suggestion
#!/bin/bash
poetry run gunicorn --bind 0.0.0.0:$PORT "todo_app.app:create_app()" 
```

Reply:
all good thanks! fixed.